### PR TITLE
feat: enable stream for CEA(js/ts)

### PR DESCRIPTION
**Pull request by QinghuiMeng-M**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/29180531/

### CODE REVIEW

File: templates/js/custom-copilot-assistant-new/src/app/app.js.tpl [hunk 1]
```diff
@@ -22,6 +22,9 @@ const model = new OpenAIModel({
 
   useSystemMessages: true,
   logRequests: true,
+  {{#CEAEnabled}} 
+  stream: true,
```
Comment (frankqianms) [inline templates/js/custom-copilot-assistant-new/src/app/app.js.tpl:26]: current teams-ai version is 1.1.0 in package.json. Does this version support stream?

Comment (kuojianlu) [reply templates/js/custom-copilot-assistant-new/src/app/app.js.tpl:26]: Agree we can upgrade the teams-ai version in package.json BTW

Comment (QinghuiMeng-M) [reply templates/js/custom-copilot-assistant-new/src/app/app.js.tpl:26]: in package-lock.json, the dependencies is "@microsoft/teams-ai": "^1.1.0", it allows changes that do not modify the left-most non-zero digit in the [major, minor, patch] tuple
so in fact, npm will down teams-ai 1.5.3

Comment (kuojianlu) [reply templates/js/custom-copilot-assistant-new/src/app/app.js.tpl:26]: Yes I know, but let's upgrade it even if we use ^

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**